nodejs/eks/package.json

```diff
@@ -45,6 +44,7 @@
         "@types/node": "^18.11.13",
         "@types/semver": "^7.3.10",
         "@types/which": "^1.3.1",
+        "jest": "^29.7.0",
```

Comment:
It's a dev dependency for the provider impl now. It's fully removed from the SDK.